cas/incident-report.md

```diff
@@ -1,169 +1,384 @@
+- **What didn’t go well:** [a list of things that caused the incident to have more impact than it otherwise would have, such as missing checks or unclear documentation. Each item here MUST have at least one corresponding Action Item below and SHOULD provide opportunities for others to ensure they make similar improvements if they haven’t already.]
+- **Where we got lucky:** [a list of things that went well, but which cannot be relied upon, such as early detection by an external security researcher or limited impact simply due to a small number of requests. Items here SHOULD generally also have corresponding Action Items, so that the CA Owner doesn’t have to rely on luck in the future.]
+- ** Other:** [any other type of "lesson learned" that does not otherwise fit in the above categories, e.g., internal/external circumstances or environmental conditions; discovery of problematic processes, policies, or workflows; communication gaps; resource challenges; task ownership; overlooked warning signs; underutilized tools; etc. Again, each item mentioned here MUST have at least one corresponding Action Item.]
+
+-->
 
 ### Action Items
+<!---
+
+**Expectations:**
+
+- This section MUST contain a list of remediation items that will be undertaken to ensure that similar incidents do not reoccur in the future. For example, if the Whiteboard field contains ‘revocation-delay’, the Action Items list MUST include steps reasonably calculated to prevent or reduce future revocation delays.
+- Note that it is not sufficient for these action items to simply stop this incident, they MUST create additional protections to prevent future incidents.
+- Each Action Item MUST state:
+     - **Action Item Description** [A detailed description of the action to be taken.]
+     - **Kind** [A classification of whether the action will help Prevent future incidents, Mitigate the impact of future incidents, or Detect future incidents. CA Owners are encouraged to propose action items in all three categories, with an emphasis on Prevent and Mitigate.] 
+     - **Corresponding Root Cause(s)** [The specific Root Cause that the Action intends to remediate (i.e., each problem/issue identified in the "Root Cause Analysis" and "What didn't go well" Sections MUST be mapped to at least one specific action item)]
+     - **Due Date** [A date by which the action item will be complete.]
+     - **Status** [Describe the status of the action item using either "Ongoing", "Complete", "Delayed", or "Canceled".]
+
+| Action Item | Kind    | Corresponding Root Cause(s) | Due Date   | Status |
+| ----------- | ----    | --------------------------- | ---------- | -------|
+| Example     | Prevent | Root Cause # 1              | 2025-01-19 | Value  |
 
-| Action Item | Kind | Due Date |
-| ----------- | ---- | -------- |
-| Example | Prevent | 2038-01-19 |
+-->
```

Comment:
Where do ongoing commitments belong?

Reply:
#187 describes ongoing commitments as being part of closure summaries.